ros_master_uri: "http://192.168.0.147:11311"
ros_bridge_ws_path: "/home/lab/GenDexGrasp/curobot_ros_ws/bridge_ws/ros2_bridge_ws"
